blueprint/core/blueprint_EcmascriptEngine.cpp

```diff
@@ -347,4 +347,9 @@ namespace blueprint
         return result;
     }
 
+    duk_context* EcmascriptEngine::getDuktapeContext()
```

Comment:
I'd like to keep this out of the `EcmascriptEngine` interface moving forward. My goal with that refactor was to create a generic interface for interacting with a js engine that could be implemented by any of various third party libraries. For example, I'm using Duktape here now, but I'd love to add another implementation via V8 that you could use with an optional switch.

So for that reason, if the details of the underlying impllementation leak into the interface, I think the interface should be reconsidered. For this case I don't think we need to do that, I'll comment below